### ISSUE TITLE
Ticket 2 - Tanmay Marwah

### ISSUE THREAD

**Issue by TMarwah** [TMarwah/P3Cowboys#35]
College Board Create Task Final Submission
- [x] Create parameter check that has 2 conditionals for 1 input(https://github.com/TMarwah/P3Cowboys/blob/aecc11bac8043fd3f949670a190d676f8574928d/Cowboys/Allen/minilab1.py#L3-L5)
- [x] Include [algorithm to calculate the average](https://github.com/TMarwah/P3Cowboys/blob/aecc11bac8043fd3f949670a190d676f8574928d/Cowboys/Allen/minilab1.py#L24-L32) as a second function that runs.
- [x] Make sure the [calls to the code](https://github.com/TMarwah/P3Cowboys/blob/aecc11bac8043fd3f949670a190d676f8574928d/Cowboys/Allen/app.py#L29-L36) are clear
- [x] Put code into [pdf](https://docs.google.com/document/d/1P4ZV5MtYi7aQtmfHLTz5zEiT3iUrVaMbOIiLVMIpNYw/edit?usp=sharing) for submission
- [x] Make final edits to writing portion on [google slide](https://docs.google.com/presentation/d/1JQsigun7J7wW9mlStOBqjzG9XE9q3AmJkTopraeY3sY/edit?usp=sharing)

**Comment by TMarwah** [comment]
I believe that my create task came out very well and highlights my proficiency in coding very well. In terms of test prep, I feel very secure with the practice we have done up until now, and I plan to complete at least one more full exam before the actual test date. Along with that exam, I will try to do my own personal reading to better understand the scope required for the exam

**Comment by BillyCherres** [comment]
I checked Tanmays links and they show evidence of completion and shows thorough work.

**Comment by jm1021** [comment]
Yes, you college board work look well done.  It has a professional flare.  Hope you do well.  3/3 +2 for early
